File: AtividadeHerancaJava/AtividadeHerancaJava/Aluno.java
public class Aluno extends Pessoa{
    private String Curso;
    private String Estudo;

    public Aluno(String Nome, int Idade, String Curso, String Estudo){
        super(Nome, Idade);
        this.Curso = Curso;
        this.Estudo = Estudo;
    }
    @Override
    public void estudar(){
        System.out.println("====ALUNO====");
        System.out.println("Nome:" + Nome);
        System.out.println("Idade:" + Idade);
        System.out.println("Curso:" + Curso);
        System.out.println("Estudo:" + Estudo);
    }
}

File: AtividadeHerancaJs/Aluno.js
class Aluno extends Pessoa{
    constructor(nome, idade, curso){
        super(nome, idade);
        this.curso=curso;
    }

    estudar(){
        return`
        Nome:${this.nome}
        Idade:${this.idade}
        Curso:${this.curso}
        `;
    }
}

const pessoa1 = new Pessoa(
    "Eduardo",
    17
)

const aluno1 = new Aluno(
    "Eduardo",
    17,
    "Técnico em Desenvolvimento de Sistemas"
)